src/collections/_documentation/platforms/android/index.md

```diff
@@ -478,15 +490,15 @@ dependencies {
 }
 ```
 
-If you want to use the SDK with the NDK but you still want to support the devices on the API level lower than 21 you can update your manifest as follows:
+If you want to use the SDK with the NDK but you still want to support the devices on the API level lower than 16 you can update your manifest as follows:
 
 ```xml
     <!-- Merging strategy for the imported manifests -->
       <uses-sdk
             tools:overrideLibrary="io.sentry.android" />
 ```
 
-With these changes, the NDK integration is going to be used only on the devices with the API level ≥ 21 and the rest of the devices with API level ≥14, but ≤21 will use just the SDK.
+With these changes, the NDK integration is going to be used only on the devices with the API level ≥ 16 and the rest of the devices with API level ≥14, but ≤16 will use just the SDK.
```

Comment:
I tried making this a bit more digestible. Hoping it's still valid.
```suggestion
With these changes, the NDK integration will be used only on:
- the devices with the API level ≥ 16  
- the rest of the devices with API level ≥ 14

Any device ≤ 16 will use the SDK.
```

Reply:
we (@fdrabek too) simplified a little bit, hopefully, it's good to go :)